src/control/dmg/storage.go

```diff
@@ -66,17 +66,19 @@ func (s *ScanStorCmd) Execute(args []string) error {
 }
 
 // FormatStorCmd is the struct representing the format storage subcommand.
-type FormatStorCmd struct{}
+type FormatStorCmd struct {
+	Force bool `short:"f" long:"force" description:"Perform format without prompting for confirmation"`
+}
 
 // run NVMe and SCM storage format on all connected servers
-func formatStor() {
+func formatStor(force bool) {
 	fmt.Println(
 		"This is a destructive operation and storage devices " +
 			"specified in the server config file will be erased.\n" +
 			"Please be patient as it may take several minutes.\n" +
 			"Are you sure you want to continue? (yes/no)")
 
-	if getConsent() {
+	if getConsent(!force) {
```

Comment:
Instead of adding a new and potentially confusing parameter to getConsent(), why couldn't you just decide whether or not getConsent() even needs to be called?

e.g.
```suggestion
	if !force && getConsent() {
```

Reply:
yes consider that initially but didn't find the force parameter confusing, will make the change

Reply:
I think the correct behaviour would be if force || getConsent

Reply:
Ah, right. Yes, that makes more sense. Looks good to me!